src/Operation/DropSearchIndex.php

```diff
@@ -53,11 +54,13 @@ public function __construct(private string $databaseName, private string $collec
     /**
      * Execute the operation.
      *
+     * TODO PHP 8.2: Change return type to null
      * @see Executable::execute()
      * @throws UnsupportedException if write concern is used and unsupported
      * @throws DriverRuntimeException for other driver errors (e.g. connection errors)
+     * @return null
      */
-    public function execute(Server $server): void
+    public function execute(Server $server): mixed
```

Comment:
The void type hint is invalid, as the interface has a `mixed` type. Since `mixed` is a type that can't combined with other types, but also doesn't include `void`, we have to use `mixed` for now. We can change this when upgrading to PHP 8.2, as that version introduces `null` as a standalone type. If we were to make this class final, we could make the change in an upcoming minor version, especially as the method only ever returns `null`.

Reply:
Would it be preferable to just remove the return type from `Executable::execute()` interface? This interface has always been internal and doesn't really serve any purpose within the library.

It is extended by Explainable, but that also seems unnecessary since the Explain operation only depends on `Explainable::getCommandDocument()`.

For context: Executable was created in a different time where I was adamant about everything having an interface ([PHPC-378](https://jira.mongodb.org/browse/PHPC-378)). Nine years later, I think we'd have done better to embrace the [YAGNI](https://en.wikipedia.org/wiki/You_aren't_gonna_need_it) philosophy earlier on.

Reply:
Sounds good. I've created PHPLIB-1519 to track the deprecation and PHPLIB-1520 for the removal of the `Executable` interface. In the meantime, I've removed the native type from `Executable::execute` in favour of a `mixed|void` type hint in the docblock. The `execute` methods of operations that would previously return `void` have been changed back to that type instead of trying to returning something semi-useful.